src/main/java/io/temporal/samples/dsl/models/ActivityInvocation.java

```diff
@@ -0,0 +1,60 @@
+package io.temporal.samples.dsl.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Strings;
+import io.temporal.activity.ActivityOptions;
+import io.temporal.workflow.ActivityStub;
+import io.temporal.workflow.Workflow;
+import java.time.Duration;
+import java.util.Map;
+
+public class ActivityInvocation {
+  public String name;
+  public String[] arguments;
+  public String result;
+
+  @JsonCreator
+  public ActivityInvocation(
+      @JsonProperty("name") String name,
+      @JsonProperty("arguments") String[] arguments,
+      @JsonProperty("result") String result) {
+    this.name = name;
+    this.arguments = arguments;
+    this.result = result;
+  }
+
+  public Void execute(Map<String, String> bindings) {
+    String[] args = makeInput(this.arguments, bindings);
+    // Class.forName(name);
+
+    ActivityStub stub =
+        Workflow.newUntypedActivityStub(
+            ActivityOptions.newBuilder()
+                .setStartToCloseTimeout(Duration.ofMinutes(1))
+                .setTaskQueue("dsl")
+                .build());
+
+    // Warning activity task takes in non-varargs call of varargs method with inexact argument type
+    // for last parameter;
+    // String results = stub.execute(name, String.class, args);
+    //                                                   ^
+    // cast to Object for a varargs call
+    // cast to Object[] for a non-varargs call and to suppress this warning
+    String results = stub.execute(name, String.class, args);
```

Comment:
At this point, it executes but it doesn't the activity never returns.

Here is the stack trace
![image](https://user-images.githubusercontent.com/16171726/94291677-c6e9eb80-ff10-11ea-9ec8-11b21573dc66.png)

![image](https://user-images.githubusercontent.com/16171726/94291712-d1a48080-ff10-11ea-97a5-f7ae98b31722.png)
![image](https://user-images.githubusercontent.com/16171726/94291742-d832f800-ff10-11ea-85d9-642593b6b9b1.png)

Reply:
cc: @mfateev